internal/pally/metric.go

```diff
@@ -66,6 +68,23 @@ type GaugeVector interface {
 	MustGet(...string) Gauge
 }
 
+// Latencies approximates a latency distribution with a histogram.
+//
+// Latencies are exported to both Prometheus and Tally using their native
+// histogram types.
+type Latencies interface {
+	Observe(time.Duration)
```

Comment:
(non-blocking)  Might be nice to have an indication that the durations passed in here should not be changed.  I can see people shooting themselves in the foot by adjusting these values based on their metric "unit"